File: Ejercicios/Ejercicio2/2.py
class Ventas:
    def __init__(self):
        self.nombre_cliente = ""
        self.subtotal = 0.0
        self.igv = 0.18
        self.total = 0.0

    def set_venta(self, nombre, sub):
        self.nombre_cliente = nombre
        self.subtotal = sub
        self.igv = self.subtotal * 0.18
        self.total = self.subtotal + self.igv

    def MostrarVenta(self):
        print("Cliente:", self.nombre_cliente)
        print("Subtotal:", self.subtotal)
        print("IGV:", self.igv)
        print("Total:", self.total)

ventas = [Ventas() for _ in range(2)]

for venta in ventas:
    nombre = input("Ingrese nombre del cliente: ")
    subtotal = float(input("Ingrese subtotal: "))
    venta.set_venta(nombre, subtotal)

print("Ventas ingresadas:")
for i, venta in enumerate(ventas):
    print("Venta", i + 1, ":")
    venta.MostrarVenta()
    print()

File: Ejercicios/Ejercicio2/2.cpp
#include <iostream>
#include <string>

using namespace std;

class Ventas 
{
private:
    string nombreCliente;
    float subtotal;
    float igv; 
    float total;

public:

    Ventas() 
    {
        nombreCliente = "";
        subtotal = 0.0;
        igv = 0.18; 
        total = 0.0;
    }


    void setVenta(string nombre, float sub) 
    {
        nombreCliente = nombre;
        subtotal = sub;
        igv = subtotal * 0.18; 
        total = subtotal + igv; 
    }


    void mostrarVenta() 
    {
        cout << "Cliente: " << nombreCliente << endl;
        cout << "Subtotal: " << subtotal << endl;
        cout << "IGV: " << igv << endl;
        cout << "Total: " << total << endl;
    }
};

int main() 
{
    Ventas ventas[10]; 

    for (int i = 0; i < 2; ++i) 
    {
        string nombre;
        float subtotal;

        cout << "Ingrese nombre del cliente: ";
        getline(cin, nombre);
        cout << "Ingrese subtotal: ";
        cin >> subtotal;

        ventas[i].setVenta(nombre, subtotal);
    }

    cout << "Ventas ingresadas:" << endl;
    for (int i = 0; i < 2; ++i) {
        cout << "Venta " << i + 1 << ":" << endl;
        ventas[i].mostrarVenta();
        cout << endl;
    }

    return 0;
}
